frontend/src/components/bullet_chooser/evidence_type_chooser.tsx

```diff
@@ -10,7 +10,7 @@ export type EvidenceTypeOption = BulletProps & {
   id: SupportedEvidenceType
 }
 
-export const supportedEvidenceTypes: Array<EvidenceTypeOption> = [
+export const supportedEvidenceCount: Array<EvidenceTypeOption> = [
```

Comment:
Was this an intentional renaming? Seems kind of odd.

Reply:
yeah - I came across this variable and got confused - I thought it was referring to the type of evidence, and not the count for each type (or something like that - I mainly remember being slightly confused and having to look at it for a minute; it was early last week). Do you prefer the former name? If so I can change it back